hircluster.c

```diff
@@ -220,20 +220,20 @@ static int cluster_reply_error_type(redisReply *reply) {
 
     if (reply->type == REDIS_REPLY_ERROR) {
         if ((int)strlen(REDIS_ERROR_MOVED) < reply->len &&
-            strncmp(reply->str, REDIS_ERROR_MOVED, strlen(REDIS_ERROR_MOVED)) ==
-                0) {
+            memcmp(reply->str, REDIS_ERROR_MOVED,
+                   strlen(REDIS_ERROR_MOVED)) == 0) {
```

Comment:
Fix clang format issue
```suggestion
            memcmp(reply->str, REDIS_ERROR_MOVED, strlen(REDIS_ERROR_MOVED)) ==
                0) {
```